src/components/checkbox/checkbox-test.stories.tsx

```diff
@@ -1,19 +1,24 @@
-import { useState } from "react";
-import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import React, { useState } from "react";
 import { action } from "@storybook/addon-actions";
 
 import specialCharacters from "../../__internal__/utils/argTypes/specialCharacters";
-import { Checkbox } from ".";
+import { Checkbox, CheckboxProps } from ".";
 
-<Meta
-  title="Checkbox/Test"
-  parameters={{
+interface StoryProps {
+  labelSpecialCharacters: string;
+  fieldHelpSpecialCharacters: string;
+  labelHelpSpecialCharacters: string;
+}
```

Comment:
```suggestion
interface StoryProps {
  labelSpecialCharacters?: string;
  fieldHelpSpecialCharacters?: string;
  labelHelpSpecialCharacters?: string;
}
```
As you are setting these to `undefined` in `Default.args` I think we should make these props optional